hare/algorithm.go

```diff
@@ -223,7 +223,7 @@ PreRound:
 	}
 	proc.preRoundTracker.FilterSet(proc.s)
 	if proc.s.Size() == 0 {
-		proc.Event().Error("Fatal: PreRound ended with empty set", log.Uint64("layer_id", uint64(proc.instanceId)))
+		proc.Event().Error("Fatal: PreRound ended with empty set", log.LayerId(uint64(proc.instanceId)))
```

Comment:
why is this a part of this pr?

Reply:
I have some minor LOG changes well because they are minor and I didn't want to miss them
One was important for Barak the other is something Yuval asked me to change if I encounter it
No risk in the above